src/app/api/exercise/[exerciseId]/route.ts

```
@@ -0,0 +1,37 @@
+import { NextResponse } from 'next/server';
+import prisma from '@/lib/prisma';
+
+// --- Standard Response Helpers ---
+const successResponse = (data: any, status = 200) => {
+  return NextResponse.json({ data }, { status });
+};
+
+const errorResponse = (message: string, status = 500, details?: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)